### ISSUE TITLE
Inform the user if the zone file on disk does not exist.

### ISSUE THREAD

**Issue by jcnelson** [blockstack/blockstack-core#375]
The `blockstack update` command is able to take both a zone file as a CLI arg, as well as a path to a file on disk with the zone file data.  However, if the path does not exist, then the CLI thinks that the path itself is the literal zone file contents.  This must be fixed--the CLI should be able to detect and resolve this ambiguity (e.g. give a warning if the zone file argument looks like a path, but refers to a non-existent file).

**Comment by jcnelson** [comment]
Related, this traceback can happen when the user passes a non-standard zone file without also passing `True` for `nonstandard`.

```
Unexpected error:
Traceback (most recent call last):
  File "/usr/local/bin/blockstack", line 52, in <module>
    result = run_cli()
  File "/usr/local/lib/python2.7/dist-packages/blockstack_client/cli.py", line 398, in run_cli
    result = method(args, config_path=config_path)
  File "/usr/local/lib/python2.7/dist-packages/blockstack_client/actions.py", line 1358, in cli_update
    fqu, user_zonefile_dict, data_pubkey
  File "/usr/local/lib/python2.7/dist-packages/blockstack_client/config.py", line 943, in configure_zonefile
    urls = user_zonefile_urls(zonefile)
  File "/usr/local/lib/python2.7/dist-packages/blockstack_client/user.py", line 168, in user_zonefile_urls
    assert is_user_zonefile(user_zonefile)
AssertionError
```

**Comment by jcnelson** [comment]
Fixed in `rc-0.14.2`.
